checkexperiments.py

```diff
@@ -0,0 +1,266 @@
+                else:
+                    # check read depth:
+                    depth_flag = False
+                    if award_obj.get('rfa') == 'modENCODE':
+                        for rep in replicates_reads:
+                            if replicates_reads[rep] < min_depth['modENCODE-chip']:
+                                depth_flag = True
+                                err.write(
+                                    award_obj.get('rfa') + '\t' + \
+                                    experiment_accession + '\t' + rep + \
+                                    '\treads_count=' + str(replicates_reads[rep]) + \
+                                    '\texpected count=' + \
+                                    str(min_depth['modENCODE-chip']) + '\n')
+                                err.flush()
+                                break
+                    else:
+                        if ex['assay_term_name'] in min_depth:
+                            for rep in replicates_reads:
+                                if replicates_reads[rep] < min_depth[ex['assay_term_name']]:
+                                    depth_flag = True
+                                    err.write(
+                                        award_obj.get('rfa') + '\t' + \
+                                        experiment_accession + '\t' + rep + \
+                                        '\treads_count=' + \
+                                        str(replicates_reads[rep]) + '\texpected count=' + \
+                                        str(min_depth[ex['assay_term_name']]) + '\n')
+                                    err.flush()
+                                    break
+                    if not depth_flag:
+                        pass_audit = True
+                        try:
+                            audit_request = session.get(urljoin(
+                                url,
+                                '/' + experiment_accession + '?frame=page&format=json'))
+                            audit_obj = audit_request.json().get('audit')
+                            if audit_obj.get("ERROR"):
+                                pass_audit = False
+                        except requests.exceptions.RequestException:
+                            continue
+                        else:
+                            if pass_audit:
+                                out.write(
+                                    award_obj.get('rfa') + '\t' + \
+                                    experiment_accession + '\t' + ex['status'] + \
+                                    '\t-> submitted\t' + max(dates).strftime("%Y-%m-%d") + '\n')
+                                out.flush()
+                            else:
+                                err.write(
+                                    award_obj.get('rfa') + '\t' +
+                                    experiment_accession + '\taudit errors\n')
+                                err.flush()
+
+
```

Comment:
Too many blank lines.  Only one blank line a row in functions.  

In general, I would not leave any blank lines in a function without really good reason.  Indenting in python takes care of visually separating code blocks so we don't need to add extra spaces.

Reply:
done